### PR TITLE
Removed outdated BPG links from table

### LINKED ISSUE #88: Remove BoutrosLab.plotting.general from function table in README
Reove the `BoutrosLab.plotting.general` and `BoutrosLab.plotting.general-package` from the first row in the function table in the README and move it above the table.

### PR DESCRIPTION
**Pull request by forbiddenpersimmon**
# Description
Originally was to move `Boutros.plotting.general` and `Boutros.plotting.general.package` links from table to above it. @stefaneng pointed out that links are outdated so have elected to remove completely from README.

### Closes #88 

# Checklist

- [X] This PR does NOT contain [PHI](https://ohrpp.research.ucla.edu/hipaa/) or germline genetic data.  A repo may need to be deleted if such data is uploaded. Disclosing PHI is a [major problem](https://healthitsecurity.com/news/ucla-health-reaches-7.5m-settlement-over-2015-breach-of-4.5m).
- [X] This PR does NOT contain molecular files, compressed files, output files such as images (*e.g.* `.png`, .`jpeg`), `.pdf`, `.RData`, `.xlsx`, `.doc`, `.ppt`, or other non-plain-text files.  To automatically exclude such files using a [.gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) file, see [here](https://github.com/uclahs-cds/template-base/blob/main/.gitignore) for example.
- [X] I have read the [code review guidelines](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List).
- [X] I have set up or verified the `main` branch protection rule following the [github standards](https://confluence.mednet.ucla.edu/pages/viewpage.action?spaceKey=BOUTROSLAB&title=GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.
- [X] The name of the branch is meaningful and well formatted following the [standards](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List), using [AD_username (or 5 letters of AD if AD is too long)]-[brief_description_of_branch].
- [ ] I have added the major changes included in this pull request to the `CHANGELOG.md` under the next release version or unreleased, and updated the date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,10 +43,9 @@ Or to install the latest development version from GitHub:
 devtools::install_github("uclahs-cds/public-R-BoutrosLab-plotting-general")
 ```
 ## Function reference
-For further details on usage, parameters or to see an example, see function links.
+For further details on usage, parameters or to see an example of a specific function, see function links below.
 | **Functions** | **Description** |
 |-----------|---------|
-|[`BoutrosLab.plotting.general-package`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/BoutrosLab.plotting.general-package.html) [`BoutrosLab.plotting.general`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/BoutrosLab.plotting.general-package.html)|Boutros Lab general plotting functions|
 |[`CNA`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/CNA.html)|Copy number aberration (CNA) data from colon cancer patients|
 |[`SNV`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/SNV.html)|Single nucleotide variant (SNV) data from colon cancer patients|
```
Comment (tyamaguchi-ucla) [inline README.md:50]: Minor point but `CNA` and `SNV` are not functions and we may want to add a new table for them.

Comment (forbiddenpersimmon) [reply README.md:50]: `CNA` and `SNV` are data frames included in package?

File: README.md [hunk 2]
```diff
@@ -43,10 +43,9 @@ Or to install the latest development version from GitHub:
...
 |[`auto.axis()`](https://uclahs-cds.github.io/public-R-BoutrosLab-plotting-general/reference/auto.axis.html)|Create ideal labels and values for a given dataset (detects log scales)|
```
[the remark this answers is not in the document]
Comment (forbiddenpersimmon) [reply README.md:51]: Looks like all 3 are there, preview also shows proper table format.

File: README.md [hunk 3]
```diff
@@ -43,10 +43,9 @@ Or to install the latest development version from GitHub:
 devtools::install_github("uclahs-cds/public-R-BoutrosLab-plotting-general")
 ```
 ## Function reference
-For further details on usage, parameters or to see an example, see function links.
+For further details on usage, parameters or to see an example of a specific function, see function links below.
 | **Functions** | **Description** |
```
Comment (stefaneng) [inline README.md:47]: Can you add a blank new line before the table? (between line 46 and 47)

**Review by stefaneng — APPROVED** [review]
Looks great! Nice work.

**Pull request merged**